src/winsign/timestamp.py

```diff
@@ -30,7 +30,7 @@ class TimeStampReq(univ.Sequence):
         namedtype.NamedType("messageImprint", DigestInfo()),
         namedtype.OptionalNamedType("reqPolicy", TSAPolicyId()),
         namedtype.OptionalNamedType("nonce", univ.Integer()),
-        namedtype.NamedType("certReq", univ.Boolean(False)),
+        namedtype.NamedType("certReq", univ.Boolean(True)),
```

Comment:
I *think* we should leave `TimeStampReq` as-is (false is the default per spec), and set `req["certReq"] = True` in `get_rfc3161_timestamp` instead.
That should also make it easier eventually to replace our own implementation of these classes with importing them from `pyasn1_modules.rfc3161` (I've got a WIP local patch for that).

Reply:
I totally agree. Thanks.